docs/rules/hv/table-1.csv

```diff
@@ -0,0 +1,21 @@
+Node Type,Originates HV?,Propagates HV?,Becomes HV? (when HV Propagates to this node),Notes
+Deep N-Well,No,N/A,N/A,"[#f1]_ ,[#f7]_ "
+P-Well,No,N/A,N/A,"[#f1]_ ,[#f7]_ "
+P-Well Resistor,No,No,Yes,[#f2]_
+N-Well,No,N/A,N/A,"[#f1]_ ,[#f7]_ "
```

Comment:
There must be a space after the comma so that `[#f7]` turns into a footnote link. I think it looks nicer without the comma, as I'm modifying the theme to turn the footnote links into a superscript.

![image](https://user-images.githubusercontent.com/8130120/90263107-c6762380-de81-11ea-8c43-4f5ee421f463.png)

![image](https://user-images.githubusercontent.com/8130120/90263202-e7d70f80-de81-11ea-99fa-9528c77aaae7.png)

Reply:
Fixed.